### PR TITLE
Check coldcard version and request user to upgrade

### LINKED ISSUE #1343: Edit Coldcard descriptor registration error message
In the descriptor registration on the signing devices step, we currently don't check the firmware version of a Coldcard to tell the user that miniscript is supported only on the Edge one.

The temporary quick fix would be to add for all registration failures happening on Coldcard, independent from the firmware version, an informative note. The error message if the device is Coldcard would then become:

"Failed to register descriptor. Note: the Edge firmware is currently required for Miniscript support"

Is it possible to insert it in the v7 release? If not, I will modify this issue to properly check the firmware version in the following release (which is already WIP if I understood correctly).

### PR DESCRIPTION
**Pull request by edouardparis**
Close #1343

### CODE REVIEW

**Comment by pythcoiner** [comment]
image
the message is the same wether we choose PSWSH or taproo, maybe if taproot is selected we should ask for version 6.3.3?

**Review by jp1ac4 — APPROVED** [review]
utACK 7371c9df40e4201876c38f4dad11fa92ec1396ce.

File: gui/ui/src/component/hw.rs [hunk 1]
```diff
@@ -348,6 +348,41 @@ pub fn unsupported_hardware_wallet<'a, T: 'a, K: Display, V: Display>(
...
+    kind: K,
+    version: Option<V>,
+    requested_version: S,
+) -> Container<'a, T> {
+    container(
+        row(vec![
+            column(vec![
+                text::p1_bold("Unsupported firmware version").into(),
+                text::p1_regular(format!(
+                    "Install version superior or equal to {}",
```
Comment (jp1ac4) [inline gui/ui/src/component/hw.rs:361]: nit: I think it would sound more natural to say "Install version {} or later".

**Comment by edouardparis** [comment]
> the message is the same wether we choose PSWSH or taproo, maybe if taproot is selected we should ask for version 6.3.3?

We avoid to add more logic for now, it is past rc and goes straight into release.

**Comment by pythcoiner** [comment]
tACK 7371c9df

**Review by jp1ac4 — APPROVED** [review]
utACK https://github.com/wizardsardine/liana/commit/7124e09b7555f46a9c16e265eed4bf247ea8326c.

**Review by edouardparis — COMMENTED** [review]
Self-ACK 7124e09b7555f46a9c16e265eed4bf247ea8326c

**Pull request merged**